imageroot/actions/set-nethvoice-admin-password/10set_password

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import json
+import os
+import sys
+import agent
+import subprocess
+
+mariadb_root_passwd = os.environ['MARIADB_ROOT_PASSWORD']
+
+request = json.load(sys.stdin)
+
+if 'nethvoice_admin_password' in request:
+    # Make sure MariaDB service is started
+    result = subprocess.run(["systemctl", "--user", "enable", "--now", "mariadb.service"], stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
```

Comment:
this can be done on a previous separeate steps of the action,  to simplify the steps code